### ISSUE TITLE
Add accessibility link to POD footer

### ISSUE THREAD

**Issue by corylown** [pod4lib/aggregator#1014]
From Astrid:

Since it is a multi-institution project, we are using this language (rather than linking to the Stanford accessibility site). Thank you for taking care of this!:
If you cannot access content or use features on this website due to a disability, please [report your accessibility issue](mailto:library-accessibility-contact@lists.stanford.edu).
'report your accessibility issue' links to: [library-accessibility-contact@lists.stanford.edu](mailto:library-accessibility-contact@lists.stanford.edu)
